index-handlers/src/main/java/no/sikt/nva/nvi/index/model/document/NviCandidateIndexDocument.java

```diff
@@ -77,6 +79,7 @@ public record NviCandidateIndexDocument(@JsonProperty(CONTEXT) URI context,
     private static final String REPORT_PENDING_VALUE = "?";
     private static final String REPORT_APPROVED_VALUE = "J";
     private static final String REPORT_DISPUTED_VALUE = "T";
+    private static final String UNSUPPORTED_LANGUAGE = "Annet språk";
```

Comment:
If this is ISO-639-3 und, it is probably "Udefinert språk", but "Annet" functions equally well in context, I guess.